frontend/public/locales/fr/renew-adult.json

```diff
@@ -61,5 +61,33 @@
       "phone-number-alt-valid": "Numéro de téléphone invalide",
       "phone-required": "Entrer un numéro de téléphone"
     }
+  },
+  "confirm-email": {
+    "page-title": "Adresse courriel",
+    "add-email": "Une adresse courriel à jour permet au gouvernement du Canada et à la Sun Life de communiquer avec vous au sujet de votre adhésion au Régime canadien de soins dentaires. Vous trouverez l'adresse courriel que nous avons dans nos dossiers dans la lettre de Service Canada.",
+    "add-or-update": {
+      "legend": "Souhaitez-vous mettre à jour ou ajouter votre adresse courriel?",
+      "help-message": "L'adresse courriel que nous avons au dossier se trouve également sur la lettre de renouvellement que vous avez reçue de Service Canada."
```

Comment:
Looks like the English version is missing for this text, unless it's not needed.